packs/mmonit/actions/list_hosts.py

```diff
@@ -0,0 +1,14 @@
+from lib.mmonit import MmonitBaseAction
+
+
+class MmonitListHosts(MmonitBaseAction):
+    def run(self, host_id):
+        self.login()
```

Comment:
If you wanted to save a couple of lines of repeated code across the actions, you could create a custom context manager class which logs in on `__enter__` and logs out on `__exit__` and then do something like this:

```python
with MoninSession() as session:
    session.get(...)
```